lib/src/tree.dart

```diff
@@ -120,12 +120,24 @@ class SimpleSelectorSequence extends TreeNode {
   bool get isCombinatorTilde => combinator == TokenKind.COMBINATOR_TILDE;
   bool get isCombinatorDescendant =>
       combinator == TokenKind.COMBINATOR_DESCENDANT;
-
-  String get _combinatorToString => isCombinatorDescendant
-      ? ' '
-      : isCombinatorPlus
-          ? ' + '
-          : isCombinatorGreater ? ' > ' : isCombinatorTilde ? ' ~ ' : '';
+  bool get isCombinatorDeep => combinator == TokenKind.COMBINATOR_DEEP;
+
+  String get _combinatorToString {
+    switch (combinator) {
+      case TokenKind.COMBINATOR_DEEP:
+        return ' /deep/ ';
```

Comment:
Should >>> be the default since that's the only deep operator for combinators in the latest spec?